app/components/register.jsx

```diff
@@ -106,7 +106,7 @@ export class Register extends Component {
                   </h2>
                   <div className="form-group">
                     <label htmlFor="dex_title">Title</label>
```

Comment:
thoughts on adding the url preview here too? not really to inform the user about conflicting urls, but just cause its cool looking